migrator/BUILD

```diff
@@ -63,7 +70,17 @@ native_java_libraries(
     native_libraries_deps = [
         "//:typedb",
         "//concept:concept",
-        "//logic:logic"
+        "//logic:logic",
+        "//rocks:rocks",
```

Comment:
awkward to depend on rocks - should we be putting the APIs on the Transaction interface that we require?